_config/ispn.yml

```diff
@@ -949,3 +949,15 @@ sb_starter:
   master:
     zip-url: https://github.com/infinispan/infinispan-spring-boot/archive/master.zip
     title: /infinispan-spring-boot/master/documentation/asciidoc/titles/spring_boot_starter.html
+
+# -----------------------------
+# 8. Simple Tutorials
+# -----------------------------
+
+ispn_tutorials:
+  # Tutorials should always come from master. Latest and greatest.
+  master:
+    # Test zip for now...
+    zip-url: https://github.com/oraNod/infinispan-simple-tutorials/archive/ISPN-10375.zip
```

Comment:
https://github.com/infinispan/infinispan-simple-tutorials/pull/82 needs merging first. then we can update the zip location.